### PR TITLE
2.0-preview branch

### PR DESCRIPTION
**Pull request by bicknellr**
~~Currently failing an accessibility test because of GoogleChrome/accessibility-developer-tools#312.~~

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -25,7 +25,9 @@
     <script>
 
       WCT.loadSuites([
-        'paper-listbox.html'
+        'paper-listbox.html?wc-shadydom=true&wc-ce=true',
+        'paper-listbox.html?dom=shadow',
+        'paper-listbox.html?wc-shadydom=true&wc-register=true'
```
Comment (e111077) [inline test/index.html]: this line is not needed

File: test/paper-listbox.html [hunk 2]
```diff
@@ -40,15 +40,18 @@
...
+        setup(function(done) {
           listbox = fixture('basic');
-        });
 
-        test('selected item has an appropriate className', function(done) {
-          Polymer.Base.async(function() {
-            assert(listbox.selectedItem.classList.contains('iron-selected'));
+          // Wait for the initial items update.
+          listbox.addEventListener('iron-items-changed', function onIronItemsChanged() {
+            listbox.removeEventListener('iron-items-changed', onIronItemsChanged);
```
Comment (e111077) [inline test/paper-listbox.html]: same iron-selector question

File: test/paper-listbox.html [hunk 3]
```diff
@@ -40,15 +40,18 @@
...
             done();
-          }, 1);
+          });
+        });
+
+        test('selected item has an appropriate className', function() {
+          assert(listbox.selectedItem.classList.contains('iron-selected'));
```
Comment (e111077) [inline test/paper-listbox.html]: 👍 👍 👍 👍

**Comment by e111077** [comment]
also the README has a `##<paper-listbox>` without a space between the hashes

**Comment by e111077** [comment]
pretty pretty please add a demo with it using `attr-for-selected`

File: test/paper-listbox.html (hunk 2 above)
Comment (e111077) [inline test/paper-listbox.html]: 👍

**Comment by e111077** [comment]
demo and readme optional (as we don't know what we're doing with the readme). just missing test/index.html fix and all is ✅

File: bower.json [hunk 4]
```diff
@@ -18,18 +18,35 @@
   "homepage": "https://github.com/PolymerElements/paper-listbox",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.1.0",
-    "iron-behaviors": "PolymerElements/iron-behaviors#^1.0.0",
-    "iron-collapse": "PolymerElements/iron-collapse#^1.0.0",
```
Comment (bicknellr) [inline bower.json]: This disappeared, but should have been moved to `devDependencies`.

Comment (bicknellr) [inline bower.json]: Actually, the demo doesn't actually use this element but I do need to remove the import.

Comment (bicknellr) [inline bower.json]: fixed

**Comment by bicknellr** [comment]
@e111077, there's a new demo using `attr-for-selected` and I cleaned up the rest of the demo page so it looks more like our other ones. I left the README alone for now.

**Comment by bicknellr** [comment]
(Retriggering tests because of changes to iron-menu-behavior.)

**Pull request merged**